pyClarion/base/components.py

```diff
@@ -244,6 +249,7 @@ def __init__(self, features: Tuple[feature, ...]) -> None:
         self._features = features
 
     def __setattr__(self, name, value):
+        """ignore if name is no in _config"""
 
```

Comment:
Should we keep `type(self)._config` or do something else? Perhaps, instead of that, we can try to use name mangling (`__config` instead of `_config` class var; not sure how it will behave though). Check out name mangling in python.